src/components/tables/Table.tsx

```diff
@@ -247,6 +249,7 @@ export class Table extends React.Component<ITableProps> {
             )
             || currentTableCompositeState.from !== nextTableCompositeState.from
             || currentTableCompositeState.to !== nextTableCompositeState.to
+            || currentTableCompositeState.data.allIds.length > nextTableCompositeState.data.allIds.length

```

Comment:
Should probably be `|| currentTableCompositeState.data.allIds.length !== nextTableCompositeState.data.allIds.length`

Reply:
the goal is to check for deletion only and let addition to avoid trigering sort and filter logic on newly added data